### DISCUSSION TITLE
When will version 9 be released?

### DISCUSSION THREAD

**Discussion by vuslatx** [d2phap/ImageGlass#1303] [General] [answered]
When will version 9 be released?

Thanks.

**Comment by d2phap** [comment]
Unluckily, I was sick in the whole February. It's still in progress, the current code does not meet minimum requirements for the first beta release. I will try to have a first beta this March.

**Reply by vuslatx** [reply]
> Unluckily, I was sick in the whole February. It's still in progress, the current code does not meet minimum requirements for the first beta release. I will try to have a first beta this March.

I hope you'll get better soon, God give healing..

**Reply by d2phap** [reply]
thanks! I'm fine now 😊

**Reply by vuslatx** [reply]
Has the beta been released?  I have'nt seen yet.

**Comment by d2phap** [comment, marked as answer]
@vuslatx It's just released a moment ago!
This is version 9.0 beta 1, rewritten from scratch with big improvements and missing features.

https://github.com/d2phap/ImageGlass/releases/tag/9.0.1.514
